packages/react/src/themes/teams-high-contrast/components/RadioGroup/radioGroupItemVariables.ts

```diff
@@ -0,0 +1,15 @@
+export default (siteVars: any) => {
+  return {
+    labelColorDefault: siteVars.white,
```

Comment:
can you create an interface for this and reuse it in the styles file where you use it?
there are examples in the other components if needer